test/e2e/framework/test_context.go

```diff
@@ -128,8 +128,8 @@ type TestContextType struct {
 	AllowedNotReadyNodes     int
 	CleanStart               bool
 	// If set to 'true' or 'all' framework will start a goroutine monitoring resource usage of system add-ons.
-	// It will read the data every 30 seconds from all Nodes and print summary during afterEach. If set to 'master'
-	// only master Node will be monitored.
+	// It will read the data every 30 seconds from all Nodes and print summary during afterEach. If set to 'controlplane'
```

Comment:
This comment may be inaccurate now if setting to 'controlplane' isn't an option yet.